ehr/src/org/labkey/ehr/utils/TriggerScriptHelper.java

```diff
@@ -124,6 +124,7 @@
 
     //NOTE: consider moving these to SharedCache, to allow them to be shared across scripts, yet reset from admin console
     private Map<Integer, String> _cachedAccounts = new HashMap<>();
+    private static Map<String, String> _centerCustomProps;
```

Comment:
This needs some comments, both for its purpose and what properties are currently supported. I recommend initializing to an empty map and annotating at @NotNull so you don't have to null check when you use it.